lib/tasks/evm_automate.rake

```diff
@@ -32,6 +34,49 @@ module EvmAutomate
     end
   end
 
+  def self.import_git_repo(options)
+    # TODO: One url per tenant or user
+    gr = GitRepository.find_or_create_by(:url => options['url'])
+    if options['userid'] && options['password']
+      if gr.authentications.first
+        gr.authentications.first.update_attributes(options.slice(*AUTH_KEYS))
```

Comment:
This will emit 2 queries against the authentications table.  Additionally, you should probably check for "default" instead of the first one because in the future if you add a second one, this will be hard to narrow down.